inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl>Demo INZ Nethui Ticket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
+	<Description>
+		<en>INZ Nethui Ticket</en>
+		<nl></nl>
+	</Description>
+	<IssueURL>
+		<en></en>
+		<nl></nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="event">
+			<Name>
+				<en>Event</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="eventDate">
+			<Name>
+				<en>Event date</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event date</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="ticketType">
+			<Name>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="fullName">
+			<Name>
+				<en>Full name</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Full name</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="email">
+			<Name>
+				<en>E-mail</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>E-mail</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="organisationName" required="false">
```

Comment:
Do we need an org name for a ticket?

Reply:
The organisation name is asked on the website, but I can remove from the ticket. But could be used inside (or online). For instance: "May I see your ticket? We are providing X for attendees from Y organisation.".

Reply:
The event would have these details in their database anyway all they need to to match the disclosed ticket number and they can work out the other information for the attendee.